draft-ietf-mops-streaming-opcons.md

```diff
@@ -219,6 +288,46 @@ Presentations:
+
+As noted elsewhere in this document, the QUIC protocol encrypts almost all of its transport parameters, and all of its payload, so any intermediaries that network operators may be using to perform analytics or even participate in current HTTP conversations will not work for HTTP/3 without making changes to their networks.
+
+As noted in {{tcp-behavior}}, there is increasing interest in transport protocol behaviors that responds to delay measurements, instead of responding to packet loss. These behaviors may deliver improved user experience, but in some cases have not responded to sustained packet loss, which exhausts available buffers along the end-to-end path that may affect other users sharing that path. The standardized QUIC protocol includes a congestion-control mechanism {{I-D.ietf-quic-recovery}} that is loss-based, and is intended to have roughly the behavior of TCP NewReno {{RFC6582}}, but the signals QUIC provides for congestion control are intended to be generic, and a sender can unilaterally chose a different algorithm to use, which might be loss-based, like CUBIC {{RFC8312}}, a delay-based congestion controller like COPA or BBR, or something completely different. 
+
+We do have experience with deploying new congestion controllers without melting the Internet (CUBIC is one example), but the point mentioned in {{tcp-behavior}} about TCP being implemented in operating system kernels is also different with QUIC. Although QUIC can be implemented in operating system kernels, one of the design goals when this work was chartered was "QUIC is expected to support rapid, distributed development and testing of features", and to meet this expectation, many implementers have chosen to implement QUIC in user space, outside the operating system kernel, and to even distribute QUIC with applications.
+
+The decision to deploy a new version of QUIC is relatively uncontrolled, compared to other widely used transport protocols, and "a new version of QUIC" can include "a new congestion controller", so QUIC protocol behavior can change quickly, without much notice. At IETF 105, Christian Huitema and Brian Trammell presented a talk on "Congestion Defense in Depth" {{CDiD}}, that explored potential concerns about new QUIC congestion controllers being broadly deployed without the testing and instrumentation that current major content providers routinely include. The sense of the room at IETF 105 was that the current major content providers understood what is at stake when they deploy new congestion controllers, but this presentation, and the related discussion in TSVAREA minutes from IETF 105 ({{tsvarea-105}}, are still worth a look from new and rapidly growing content providers. 
```

Comment:
I think there is a sublte point that is hard to make here. I'd be surprised if operators rolled a QUIC version just to deploy a different CC - that is the beauty of sender unilateral descisions. However, what we might see is that protocol evolution can support new or better approaches to CC. For example, Christians's proposed one-way delay measurement extension. Tangentially, some experimenters might want to add transport feautres that need alternative CC, and it can be easier to run an experimental version of QUIC to prove a hypothesis rather than try to hook into extension mechanisms.

I really don't know best to present such nuance here. All I can suggest is to focus on concerns your target audience has, rather than nitty gritty QUIC detail.s